python/pyspark/pandas/accessors.py

```diff
@@ -752,7 +752,7 @@ def transform_batch(self, func, *args, **kwargs) -> "Series":
         See Also
         --------
         DataFrame.pandas_on_spark.apply_batch : Similar but it takes pandas DataFrame as its
-        internal batch.
+            internal batch.
```

Comment:
This is a required change to inline to the item above: `DataFrame.pandas_on_spark.apply_batch`